server/src/lib/basket.ts

```diff
@@ -143,19 +176,24 @@ async function updateFullBasket(currentUserStats: UserEmailStats) {
     two_day_streak: Boolean(computed.two_day_streak),
   };
 
-  sendRequest({
-    ...basketConfig,
-    form: data,
-  }).catch(err => console.error(err.message));
+  // @TODO: How does this handle Basket unsubscribes?
```

Comment:
That note is more about the fact that that newsletter unsubscribes happen in the newsletter management centre `https://www.mozilla.org/en-US/newsletter/existing/<basket_token>` and then there's nothing that gets sent back to the CV platform to indicate that the user has changed their preferences, so it will keep syncing to Cinchy/Acoustic to keep sending goal emails even after they unsubscribe from a newsletter. The easiest fix would be to make a call to Basket before every update to Acoustic but that seems very not-performant and I wanted to think through whether there's a better/different solution.

Reply:
We will know in Cinchy if someone has changed their preferences, as CTMS is where universally, newsletter preferences are being stored (and provided to Acoustic). Cinchy will be integrated w/ that entire ecosystem.

Reply:
@jackielu-mozilla Oh I didn't realize that, that's awesome. Thanks for clarifying!